test/sde/sde_additive_tests.jl

```diff
@@ -46,20 +46,20 @@ println("Convergence Test on MultiDimAdditive")
 dts = (1/2) .^ (7:-1:4) #14->7 good plot
 
 sim = test_convergence(dts,prob,SRA(),numMonte=5)
-@test abs(sim.𝒪est[:l2]-2) < 0.1
+@test abs(sim.𝒪est[:l2]-2) < 0.15
```

Comment:
should probably just up the number of trajectories `numMonte` instead. Hopefully that doesn't send the compute time up too high, but 0.15 is getting a little up there when SDE methods have 0.5 as the true spacings.